### DISCUSSION TITLE
what is  wrong with the variables became negative?

### DISCUSSION THREAD

**Discussion by Wenjing1018** [Open-Systems-Pharmacology/Forum#813] [Questions & Problems]
Hello, 
the specific anatomical parameters have not been modified, but there is an error during operation (as shown in the attachment).
I can't figure out where it went negative?
![image](https://user-images.githubusercontent.com/63133525/88629802-4eb6a380-d0e2-11ea-9a97-5ef087f09ee9.png)

**Comment by Yuri05** [comment]
First, please try to reduce solver tolerances as suggested:
![grafik](https://user-images.githubusercontent.com/25061876/88635811-6b3de600-d0b8-11ea-9647-a4eccbb68d23.png)

In order to change the solver settings in a simulation: go to the **Parameters** tab of the simulation and then to **Settings/SolverSettings** (make sure that **Advanced** mode is active)
![grafik](https://user-images.githubusercontent.com/25061876/88570016-a0f0b980-d03b-11ea-84b4-8e05e865d871.png)

**Comment by Wenjing1018** [comment]
@Yuri05,
I failed to solve this problem in your first way.

**Comment by Yuri05** [comment]
@Wenjing1018 Can you attach your project?

**Reply by ljiaqian2020-cloud** [reply]
I tried reducing the solver tolerances as suggested, but it still hasn't resolved my issue
